src/main/java/uk/gov/digital/ho/hocs/casework/reports/domain/mapping/OpenCasesDataMapper.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.digital.ho.hocs.casework.reports.domain.mapping;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+import uk.gov.digital.ho.hocs.casework.reports.domain.reports.OpenCasesData;
+import uk.gov.digital.ho.hocs.casework.reports.domain.reports.OpenCasesRow;
+
+@Service
+@Slf4j
+@Profile("reports")
+public class OpenCasesDataMapper {
+
+    private final UserNameValueMapper userNameValueMapper;
+
+    private final TeamNameValueMapper teamNameValueMapper;
+
+    private final StageNameValueMapper stageNameValueMapper;
+
+    private final ExemptionDatesAgeAdjustmentLookup exemptionDatesAgeAdjustmentLookup;
+
+    @Autowired
```

Comment:
Autowired shouldn't be required here because the class is annotated with a Stereotype @Service